lms/templates/public_video.html

```diff
@@ -16,3 +19,22 @@
 <meta data-rh="true" name="twitter:player:width" content="1280">
 <meta data-rh="true" name="twitter:player:height" content="720">
 </%block>
+
+<%block name="body_extra">
+<nav class="nav-links" aria-label="Learn More">
+  <div>
+    <a class="btn-default btn" href="${learn_more_url}">
```

Comment:
this was just testing out what button styles we have currently. you'll almost def need to write new css for this